parsel/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Parsel lets you extract text from XML/HTML documents using XPath
+or CSS selectors
+"""
+
+__author__ = 'Scrapy project'
+__email__ = 'info@scrapy.org'
+__version__ = '1.0.0'
+
+from parsel.unified import *
```

Comment:
what about exposing the API explicitly? `from parsel.unified import Selector, ...`